typescript/sdk/src/router/schemas.ts

```diff
@@ -12,6 +15,9 @@ export const MailboxClientConfigSchema = z.object({
   mailbox: ZHash,
   hook: ZHash.optional(),
   interchainSecurityModule: IsmConfigSchema.optional(),
+  ismFactoryAddresses: z
+    .record(ProxyFactoryFactoriesSchema.keyof(), ZHash)
+    .optional(),
```

Comment:
If i recall our convo @yorhodes, a set of Ism Factories will always be deployed with a Mailbox

Reply:
if I understand correctly - the `ismFactoryAddresses` are optional here because we don't need them in the existing places/files where we define mailbox client configs - but the crud modules will need them and core/warp will enforce these addresses are there before calling into ism/hook/warp modules?